renovate.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
We've got a Github action that will run gomod tidy -- https://github.com/trussworks/terraform-aws-ecs-service/blob/main/.github/workflows/tidy.yml. I think we can also remove that infavor of doing it directly in renovate. 

See Module Tidying in https://docs.renovatebot.com/golang/